app/views/publisher_information/_publisher_info_nav.html.erb

```diff
@@ -3,23 +3,29 @@
     aria_label: "Publisher information",
     items: [
       {
-        label: "What the Beta can and cannot do",
+        label: t("publisher_information.beta_capabilities.title"),
```

Comment:
I believe these are intentionally not using the translation key, as per https://github.com/alphagov/content-publisher/blob/master/docs/adr/0004-editing-microcopy.md section on link and button labels.

In the past we've also consciously chosen not to use translation keys as variables that can be re-used to reduce duplication. This is because we don't know if when someone edits them they want to edit one place or both. Though could be convinced it's the right thing in this case. Though if we do so we should change the test to use the translation key rather than the strings directly.

All essentially minor concerns, especially as we have very little editing of copy, but just pointing out where this may be drifting from what is consistent.

Reply:
I've removed the commit that made these changes to leave the links as static text